### DISCUSSION TITLE
Is Java library built with OpenMP?

### DISCUSSION THREAD

**Discussion by andreabrduque** [microsoft/onnxruntime#10641] [Other Q&A] [answered]
Im having memory issues using onnx Java, which I suspect is because of number of thread creations.
I saw this issue: https://github.com/microsoft/onnxruntime/issues/4093 

So wondering if the SDK is built with OpenMP ?

Thanks

**Comment by Craigacp** [comment, marked as answer]
I believe the Java bindings are no longer built with OpenMP since at least [v1.8.0](https://github.com/microsoft/onnxruntime/releases/tag/v1.8.0), and the OpenMP support has been removed from the master branch (https://github.com/microsoft/onnxruntime/pull/10032).

Are you controlling the thread creation using the inter op and intra op parameters? Or constructing the environment with a single shared thread pool?

**Reply by andreabrduque** [reply]
Using intra op parameters set to 1. My memory issue was something else.
